docs/pages/getting-started/deployment.mdx

```diff
@@ -2,29 +2,117 @@
 title: Deploy vCluster
 sidebar_label: 2. Deploy vClusters
 ---
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
 
-import DeploySegment from '../fragments/deploy-vcluster.mdx'
 
-Let's create a virtual cluster `my-vcluster` in namespace `vcluster-my-vcluster`:
+## Before You Begin
 
-<DeploySegment/>
+### Admin Permissions Not Required
 
-:::info Host Namespace
-The host namespace will be generated if it does not yet exist. No matter which namespaces we create within the virtual cluster, everything will be encapsulated inside the created host namespace that the virtual cluster runs inside.
-:::
+One of the biggest benefits of vCluster compared to other virtual cluster approaches is that it does not require any special permissions. You can spin up a virtual cluster even if you do not have a cluster-admin role and only have access to deploy applications to one specific namespace.
+
+### Namespaces
+
+Installation creates the namespace for vCluster if it does not yet exist. When you create namespaces within your virtual cluster, those resources are encapsulated within the vCluster namespace. 
+
+### Air-Gapped Clusters
+
+If you want to deploy vCluster in an air-gapped environment, set the following option in the `values.yaml` used to deploy vCluster:
 
-:::caution Air-gapped Clusters
-If you want to deploy vClusters in an air-gapped environment, you can set the following option in the `values.yaml` used to deploy vCluster:
 ```
 defaultImageRegistry: my-private-registry:5000/vcluster/
 ```
-This will tell vCluster to prepend the above image registry to all images used by vCluster, such as syncer, k3s, coredns etc. So for example `rancher/k3s:v1.22.2-k3s1` will become `my-private-registry:5000/vcluster/rancher/k3s:v1.22.2-k3s1`
+
+This tells vCluster to prepend the image registry to all images used by vCluster, such as syncer, K3s, and CoreDNS. For example, `rancher/K3a:v1.22.2-k3s1` becomes `my-private-registry:5000/vcluster/rancher/k3s:v1.22.2-k3s1`.
```

Comment:
Just saw this one:

```suggestion
This tells vCluster to prepend the image registry to all images used by vCluster, such as syncer, K3s, and CoreDNS. For example, `rancher/k3s:v1.22.2-k3s1` becomes `my-private-registry:5000/vcluster/rancher/k3s:v1.22.2-k3s1`.
```

Also 1.22 is a bit of a dated version, might want to use a more recent one.